pkg/registry/core/service/strategy.go

```diff
@@ -158,8 +149,9 @@ func (svcStrategy) Export(ctx context.Context, obj runtime.Object, exact bool) e
 	if exact {
 		return nil
 	}
-	if t.Spec.ClusterIP != api.ClusterIPNone {
-		t.Spec.ClusterIP = ""
+	//set ClusterIPs as nil - if ClusterIPs[0] != None
+	if len(t.Spec.ClusterIPs) > 0 && t.Spec.ClusterIPs[0] != api.ClusterIPNone {
+		t.Spec.ClusterIPs = nil
```

Comment:
don't we need to keep the original code (clearing `ClusterIP`) too?
(also, space after `//` in the comment. Or just remove the comment since it is just paraphrasing the code.)

Reply:
Good catch. thank you.